miniz_oxide/Cargo.toml

```diff
@@ -29,5 +33,3 @@ compiler_builtins = { version = '0.1.2', optional = true }
 # Internal feature, only used when building as part of libstd, not part of the
 # stable interface of this crate.
 rustc-dep-of-std = ['core', 'alloc', 'compiler_builtins', 'adler/rustc-dep-of-std']
-# Use std instead of alloc. This should only be used for backwards compatibility.
-no_extern_crate_alloc = []
```

Comment:
Can you leave the feature with comment that it doesn't really do anything? We currently specify it and I think that would break compilation.

Reply:
Whoops! For some reason I thought that cargo just ignored missing features. Sorted it.